ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -90,41 +90,27 @@
 
 ### BGP Neighbors
 
-| Neighbor | Remote AS | VRF |
-| -------- | --------- | --- |
+| Neighbor | Remote AS | VRF | Send-community | Maximum-routes |
+| -------- | --------- | --- | -------------- | -------------- |
 {%         for neighbor in router_bgp.neighbors | arista.avd.natural_sort %}
 {%             if router_bgp.neighbors[neighbor].remote_as is arista.avd.defined %}
```

Comment:
The logic here is broken for extra columns. We can not detect if `send_community`, `vrf`, `maximum_routes` etc should be `-` or `inherited from peer group` based on presence of `remote_as`.
I think we need to rebuild everything here.

Reply:
> The logic here is broken for extra columns. We can not detect if `send_community`, `vrf`, `maximum_routes` etc should be `-` or `inherited from peer group` based on presence of `remote_as`.

Agreed, it would also be interesting to know what happens on the VEOS in this instance.

VEOS side put aside, I think we'll need to implement a weight value in here for peer groups, and for neighbors, or set one dominant, which would override the other.